sni/templates/docs/reciprocal-altruism-in-the-theory-of-money.html

```diff
@@ -334,7 +334,7 @@ <h3>
 		</p>
 
 		<p>
-			A money system just needs to be an association of a number with each person that they can transfer to one another, and which can be used to track a person’s altruism. Money must be scarce because if it were not, then anyone could easily increase the size of their number associated with them without having to provide favors for them. Money must be difficult to counterfeit because if it were not, it that just means that it is hard to tell what the correct number associated with a person should be. Money must be fungible, divisible, and durable because then it wouldn’t act like a number at all. Finally, money must be portable because then it could not be easily transferred and would necessarily be less liquid. In a global economy, in which people do business from opposite ends of the world, money should not only be portable, but teleportable. 
+			A money system just needs to be an association of a number with each person that they can transfer to one another, and which can be used to track a person’s altruism. Money must be scarce because if it were not, then anyone could easily increase the size of their number associated with them without having to provide favors for them. Money must be difficult to counterfeit because if it were not, it that just means that it is hard to tell what the correct number associated with a person should be. Money must be fungible, divisible, and durable because then it won't act like a number at all. Finally, money must be portable because then it could not be easily transferred and would necessarily be less liquid. In a global economy, in which people do business from opposite ends of the world, money should not only be portable, but teleportable.
```

Comment:
"Money must be fungible, divisible, and durable because then it wouldn’t act like a number at all" sounds wrong but I don't think this is an improvement on second thought.

Perhaps

"Money must be fungible, divisible, and durable ~~because then~~ **or else** it wouldn’t act like a number at all"

Reply:
Let's change this to:
"Money must be fungible, divisible, and durable, or else it wouldn’t act like a number at all"